crates/sui-core/src/authority.rs

```diff
@@ -2150,6 +2162,7 @@ impl AuthorityState {
                 .epoch_start_state()
                 .protocol_version(),
         );
+        self.clear_object_per_epoch_marker_table()?;
```

Comment:
I wonder if it would be better to instead include it as part of the pruner process instead of  critical path of reconfiguration?